server/src/com/game/agarserver/communication/CommunicationListener.java

```diff
@@ -0,0 +1,44 @@
+package com.game.agarserver.communication;
+
+import com.game.agar.shared.Position;
+import com.game.agarserver.logic.Ball;
+import com.game.agarserver.logic.PacketFactory;
+import com.game.agarserver.logic.User;
+import com.game.agarserver.logic.World;
+import org.json.JSONObject;
+
+import java.util.List;
+import java.util.function.Consumer;
+
+/**
+ * Created by Wojtas on 2017-04-21.
+ */
```

Comment:
https://stackoverflow.com/questions/24370502/turning-off-created-by-stamp-when-generating-files-in-intellij